app/services/leagues/matches/pick_bans/deferring_service.rb

```diff
@@ -6,16 +6,16 @@ module DeferringService
 
         def call(pick_ban, user)
           pick_ban.transaction do
-            pick_ban.defer!(user)
-
             notify_captains!(pick_ban)
```

Comment:
This is a problem. Notifications must get sent last because they involve sending irreversible emails. If deferring fails the email would still be sent with this change.